django/tasks/utils.py

```diff
@@ -0,0 +1,35 @@
+import inspect
+import json
+from traceback import format_exception
+
+from django.utils.crypto import get_random_string
+
+
+def is_module_level_function(func):
+    if not inspect.isfunction(func) or inspect.isbuiltin(func):
+        return False
+
+    if "<locals>" in func.__qualname__:
+        return False
+
+    return True
+
+
+def json_normalize(obj):
+    """
+    Round-trip encode object as JSON to normalize types.
+    """
+    return json.loads(json.dumps(obj))
+
+
+def get_exception_traceback(exc):
+    return "".join(format_exception(exc))
```

Comment:
I'm not sure this is worth as a separated helper, as far as I saw it's used in a single place, so it can be well used there as defined (django/tasks/backends/immediate.py:7):
```
traceback="".join(format_exception(e)),
```